hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenGrantKycToAccountHandler.java

```diff
@@ -110,9 +110,9 @@ private TokenRelation validateSemantics(
             @NonNull final TokenID tokenId,
             @NonNull final WritableTokenRelationStore tokenRelStore)
             throws HandleException {
-        final var tokenRel = tokenRelStore.getForModify(accountId.accountNumOrThrow(), tokenId.tokenNum());
-        validateTrue(tokenRel.isPresent(), INVALID_TOKEN_ID);
+        final var tokenRel = tokenRelStore.getForModify(accountId, tokenId).orElse(null);
+        validateTrue(tokenRel != null, INVALID_TOKEN_ID);
```

Comment:
NAy reason why we are doing `orElse(null)` and checking null here  instead of `isPresent()`?